### DISCUSSION TITLE
Dockerize

### DISCUSSION THREAD

**Discussion by Dvalin21** [phili67/ecclesiacrm#2473] [Q&A] [answered]
Just wondering if this will be dockerized?  Thanks

**Comment by phili67** [comment]
Hi,

docker is available but only for developper.

Philippe

**Comment by Dvalin21** [comment]
I take it it wont be available for for production, docker version that is?

**Reply by phili67** [reply, marked as answer]
No it's only available for dev and for local development.

**Reply by Dvalin21** [reply]
Ok, thank you very much.
